src/toTransform.js

```diff
@@ -42,10 +42,10 @@
     if ('offset-position' in properties) {
       position = internalScope.offsetPositionAnchorParse(properties['offset-position']);
     }
-
     if (position === 'auto' || position === undefined || position === null) {
       return null;
     }
+    position = [position[0].value, position[1].value];
```

Comment:
When we have px, we need to skip the step of multiplying by offsetParent size / 100.